portfolio/src/main/webapp/index.html

```diff
@@ -1,26 +1,29 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Alex's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="myscript.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>Alex's Portfolio</h1>
-      <p>Hello! My name is Alexandre Walsh, but I go by Alex. 
-          I am a rising third-year at UCLA studying Computer Science and I'm from San Diego, CA. 
-          I was born in Barcelona, however, and, fun fact, I got to meet the president of Catalunya 
-          (Jordi Pujol) when I was 4 years old! I love exploring LA (like in the below picture, 
-          where I watched Caravan Palace perform at the Wiltern Theater) and was really looking forward to 
-          getting to explore the Mountain View area this summer, but am glad that I still get the amazing opportunity 
-          to work with Google and the Google Fi team this summer :)</p>
-    <div>
-      <img src="/images/Alex_at_the_Wiltern.jpg" style="width:432px;height:576px">
-    </div>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>Alex's Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="myscript.js"></script>
```

Comment:
You could put the script import under html tag directly for example: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_script_src

Reply:
the example looks the same as my code with the script import I believe? (<script src="demo_script_src.js"></script>). 
What do you mean?

Reply:
It is super nit. It is common to put <script> tag under <html> tag instead of <head>.